src/ripple/app/tx/impl/SetRegularKey.cpp

```diff
@@ -55,6 +56,13 @@ SetRegularKey::preflight (PreflightContext const& ctx)
     if (!isTesSuccess (ret))
         return ret;
 
+    if (ctx.rules.enabled(fixDisabledRegularKey)
+        && ctx.tx.isFieldPresent(sfRegularKey)
+        && (ctx.tx.getAccountID(sfRegularKey) == ctx.tx.getAccountID(sfAccount)))
```

Comment:
Nit: Long line. We try to keep lines under 80 characters so that folks using terminals or other small screens don't have to deal with weird wrapping. I suggest a line break after the `==`.

Reply:
Agreed. I'm one of those people. I can't explain why Vim didn't already wrap it for me.